opt/jvm-redis.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# Support for Spring Data Redis
+if [ "${DISABLE_SPRING_REDIS_URL:-}" != "true" ] &&
```

Comment:
Use `[[`, http://mywiki.wooledge.org/BashFAQ/031

Also, I'd maybe make the test "if it's `false` or `0`" instead.